src/interfaces/erc20TokenManager.sol

```diff
@@ -0,0 +1,27 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.20;
+
+import "@openzeppelin/contracts/token/ERC20/ERC20.sol";
+import "@openzeppelin/contracts/access/AccessControl.sol";
+
+contract ERC20TokenManager is ERC20, AccessControl {
```

Comment:
this file should be named like IERC20TokenManager? and it should be only interface?

Reply:
True. It will be renamed